core/ir/instr_api.h

```diff
@@ -339,7 +344,7 @@ struct _instr_t {
 
     /* Used to hold the relative offset within an instruction list when encoding. */
     size_t offset;
-};     /* instr_t */
+}; /* instr_t */
```

Comment:
clang-format change.
Need to investigate if our internal version is >= than 14.

Reply:
Manually fixed.